test/canvas.test.js

```diff
@@ -53,7 +54,7 @@ describe("Canvas", ()=>{
       expect(canvas.async).toBe(true)
 
       let promise = canvas.png
-      expect(promise).toBeInstanceOf(Promise)
+      expect(util.types.isPromise(promise))
```

Comment:
I'm not sure why the `Promise` returned by Node-API is a different class than `global.Promise`.